macosx/QuickLookExtension/Info.plist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSExtension</key>
+	<dict>
+		<key>NSExtensionAttributes</key>
+		<dict>
+			<key>QLIsDataBasedPreview</key>
+			<true/>
+			<key>QLSupportedContentTypes</key>
+			<array>
+				<string>org.bittorrent.torrent</string>
+			</array>
+			<key>QLSupportsSearchableItems</key>
+			<false/>
+		</dict>
+		<key>NSExtensionPointIdentifier</key>
+		<string>com.apple.quicklook.preview</string>
+		<key>NSExtensionPrincipalClass</key>
+		<string>PreviewProvider</string>
+	</dict>
+</dict>
+</plist>
```

Comment:
This is fairly bare-bones for a valid plist file. This is because the Xcode build setting `GENERATE_INFOPLIST_FILE` is set: it moves a lot of stuff to the Xcode project itself.

If this file is only used when building via Xcode, this warning can be ignored.

Reply:
Indeed, this file is used only for building with Xcode.

But I have [macosx/QuickLookExtension/Info.plist.in](https://github.com/transmission/transmission/pull/7213/files#diff-df62d3d0b9658d89b466e5180dd90bd7d268b94f83437e203bc2915f2c2092a7) for CMake build.

Reply:
Ah, okay.